pymyq/request.py

```diff
@@ -163,7 +170,10 @@ async def _send_request(
                 )
                 if err.status == 401:
                     raise err
-
+                if err.status == 429:
+                    _LOGGER.warning("Too many request have been made - putting a temporary pause on sending any requests for %d minutes. Headers are:%s", TOO_MANY_REQUEST_TIMEOUT/60, err.headers if err.headers else None)
+                    self._block_request_until = datetime.utcnow() + datetime.timedelta(seconds=TOO_MANY_REQUEST_TIMEOUT)
```

Comment:
I think you need to drop the `datetime` as we import `timedelta`

Reply:
appreciate it